Sources/TuistCore/Graph/Nodes/TargetNode.swift

```diff
@@ -113,3 +114,15 @@ public class TargetNode: GraphNode {
         sdkDependencies.contains(where: { $0.name == "XCTest" }) || target.product.testsBundle
     }
 }
+
+private extension PrecompiledNode {
```

Comment:
I think this can be useful in other places so I'd rather move it to the body of `PrecompiledNode`.

Reply:
I got rid of this property, because it is not needed anymore.